lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1266,6 +1266,9 @@ namespace Js
             AddFunctionToLibraryObjectWithPropertyName(globalObject, _u("emitTTDLog"), &GlobalObject::EntryInfo::EmitTTDLog, 2);
         }
 #endif
+#ifdef ENABLE_DEBUG_CONFIG_OPTIONS
+        AddFunctionToLibraryObjectWithPropertyName(globalObject, _u("chWriteTraceEvent"), &GlobalObject::EntryInfo::ChWriteTraceEvent, 1);
```

Comment:
Don't have strong feelings one way or the other. My gut is to expose this always in test builds, but I can put it under a flag if you feel strongly.